tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -113,3 +114,14 @@ def test_c2r_latest_older_unsupported_version(convert2rhel):
     # Clean up
     os.system(f"cp /tmp/__init__.py {PATH_TO_VERSION}")
     del os.environ["CONVERT2RHEL_UNSUPPORTED_VERSION"]
+
+
+def test_rhsm_error_logged(convert2rhel):
+    """
+    Test if the OSError for RHSM certificate being removed
+    is not being logged in cases the certificate is not installed yet.
+    """
+    with convert2rhel("--debug --no-rpm-va") as c2r:
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("n")
+        assert c2r.expect("DEBUG - No RHSM certificates found to be removed") == 0
```

Comment:
Looks good. Just in case I would probably add a check that would make sure that the `ERROR - OSError(2): No such file or directory` message is not shown (for regression reasons). Maybe the easiest way to do it would be to look into the `/var/log/convert2rhel/convert2rhel.log` file and check that this line is not present.

Reply:
You are right, thank you for noting that! 
Regression check included now